src/generic_ui/polymer/cloud-install.html

```diff
@@ -58,13 +63,39 @@
     <core-signals on-core-signal-open-cloud-install-dialog='{{ openLoginDialog }}'></core-signals>
 
     <!-- TODO: use core-pages rather than multiple dialogs -->
+    <uproxy-dialog layered='false' backdrop='true' id='regionDialog'>
+      <h1>Create a cloud server</h1>
+      <core-icon icon='clear' on-tap='{{ closeDialogs }}'></core-icon>
+      <p>We can help you create a uProxy cloud server in a country of your choice.</p>
+      <p>To get started, please select the region for your server. You should choose the region closest to you to minimize latency.</p>
+      <div style='text-align:center;'>
+        <paper-dropdown-menu label='Select Region'>
+          <paper-dropdown class='dropdown'>
+            <!-- TODO: get available regions from digital ocean -->
+            <core-menu class='menu' id='regionMenu' valueattr='value' selected='nyc2'>
+              <paper-item value='nyc2'>New York</paper-item>
+              <paper-item value='sfo1'>San Francisco</paper-item>
+              <paper-item value='lon1'>London</paper-item>
+              <paper-item value='ams2'>Amsterdam</paper-item>
+              <paper-item value='sgp1'>Singapore</paper-item>
+              </core-menu>
+          </paper-dropdown>
+        </paper-dropdown-menu>
+      </div>
+      <div style='text-align:center;'>
+        <uproxy-button raised on-tap='{{ regionTapped }}'>
+          <span id='buttonLogoContainer'><core-icon icon='uproxy-icons:Cloud'></core-icon></span>
+          <span id='buttonTextContainer'>Choose a region</span>
```

Comment:
Since this is the last step before uProxy creates the server, how about "Create"? Also, I think a plain button will suffice here: I only used a fancy button on the "log into DigitalOcean" screen to match the other "Connect with a friend" options.